uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-balance-tab.component.js

```diff
@@ -148,9 +148,9 @@ define([
      *
      * @param {Object} event - the component event handler.
      * @param {Object} _filters_ - The filter values to use for updating the report.
-     * @param {Object} _filters_.absence_period - the absence period id to filter by.
-     * @param {Object} _filters_.absence_type - the abence type id to filter by.
-     * @param {Object} _filters_.managed_by - the ID of the manager that is retrieving the report.
+     * @param {String} _filters_.absence_period - the absence period id to filter by.
```

Comment:
JS Doc recommends the format used here:
http://usejsdoc.org/tags-param.html#parameters-with-properties

Should I replace it?